frontend/src/Elements/components/Portability/UserData.js

```diff
@@ -23,8 +23,8 @@ class UserData extends React.PureComponent {
 
     if (processors && status.processors) {
       processorsList = processors.map(p => {
-        const s = _.find(status.processors, { id: p.id });
-        p.status = SUBJECT_DATA_STATUS[s.status];
+        const s = _.find(status.processors, { id: p.id }) || {};
+        p.status = s.status ? SUBJECT_DATA_STATUS[s.status] : 'Unconsented';
```

Comment:
I think it makes sense to move processors decoration to Subject context and do it only once - when fetch processors.

Reply:
I like the idea, will do it!